app/views/data_services/index.html.erb

```diff
@@ -2,9 +2,8 @@
 
 <% @organisations.each do |organisation| %>
   <h2 class="govuk-heading-l"><%= organisation.name %></h2>
-
   <ul class="govuk-list govuk-list--bullet">
     <%= render partial: 'data_service', collection: organisation.data_services %>
   </ul>
 
-<% end %>
+<% end %>
```

Comment:
Just a hint to avoid empty unused line-breaks: you can use -%>

https://stackoverflow.com/questions/7996695/what-is-the-difference-between-and-in-erb-in-rails